docs/book/Makefile

```diff
@@ -31,6 +33,22 @@ RELEASELINK := $(TOOLS_BIN_DIR)/mdbook-releaselink
 $(RELEASELINK): $(TOOLS_DIR)/go.mod
 	cd $(TOOLS_DIR); go build -tags=tools -o $(BIN_DIR)/mdbook-releaselink ./mdbook/releaselink
 
+MDBOOK := $(TOOLS_BIN_DIR)/mdbook
+$(MDBOOK):
+	curl -LSfs https://benmoss.github.io/trust/install.sh | sh -s -- --git rust-lang/mdBook --tag v0.4.1 --to $(TOOLS_BIN_DIR) --force
+
+MDBOOK_LINKCHECK := $(TOOLS_BIN_DIR)/mdbook-linkcheck
+$(MDBOOK_LINKCHECK):
+	curl -LSfs https://benmoss.github.io/trust/install.sh | sh -s -- --git Michael-F-Bryan/mdbook-linkcheck --tag v0.7.0 --to $(TOOLS_BIN_DIR) --force
```

Comment:
What's this doing? Seems rather something we shouldn't do here, curl and pipe to bash/sh usually raises security concerns. 

/hold

Reply:
Yes, I anticipated there might be concerns around this. We can vendor this script, but it's a script to install precompiled Rust crate published to GitHub. It's from https://github.com/japaric/trust but because of https://github.com/japaric/trust/pull/132 it's pulling from my fork.

Reply:
Updated to vendor the script, WDYT?